SparkPost/2web2ui
# SA-1490: Remove X hours remaining on test from Inbox Placement

### What Changed
 - Removed the X hours remaining on test from running tests.

### How To Test
 - Go to /inbox-placement. See that a test is still running but not longer displays 'X hours remaining on test'
